apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/oppgaver/Oppgaver.tsx

```diff
@@ -70,6 +72,7 @@ export const Oppgaver = ({
         oppgaver={paginerteOppgaver}
         oppdaterTildeling={oppdaterSaksbehandlerTildeling}
         oppdaterFrist={oppdaterFrist}
+        oppdaterStatus={oppdaterStatus}
```

Comment:
Nå er vi allerede på 3 endring av oppgave. Da kommer spørsmålet om man kanskje heller skulle slått alle 3 sammen for å redusere bloat i koden?

Reply:
Vi kunne fint ha slått funksjonene sammen i `oppgaveHandlinger.ts` og laget en finnOgOppdaterOppgave, men litt usikker på hvorvidt vi burde slå sammen `oppdaterTildeling` sammen med `oppdaterFrist` og `oppdaterStatus` da den bruker en del andre parametere for å håndtere det med saksbehandler.